src/extension/tab/hook.ts

```diff
@@ -54,7 +54,39 @@ function initializeHook() {
     clientRelay.broadcast(data);
   });
 
+  type QueryInfo = {
+    document: any;
+    source: any,
+    variables: any;
+    lastWrittenResult: any;
+    lastWrittenVars: any;
+  }
+
```

Comment:
Maybe consider moving this out of the `initializeHook` function, putting it above the `Hook` type declaration above. That way `Hook` can use this as part of its `getQueries` definition (see comment above).

Also, to help get rid of the `any`'s here, maybe something like:

```ts
import { DocumentNode, Source } from "graphql";
...
type QueryInfo = {
  document: DocumentNode;
  source?: Source,
  variables?: Record<string, any>;
  lastWrittenResult?: Record<string, any>;
  lastWrittenVars?: Record<string, any>;
}
```